### PR TITLE
[Improve] Improve streampark-console system package module service query method name base on [3.1 Naming Style]

### PR DESCRIPTION
**Pull request by VampireAchao**
## What changes were proposed in this pull request

Issue Number: see #3064  

## Brief change log

Improve streampark-console core package module service name base on [3.1 Naming Style]

1. **Just query method**
2. **Just in service**

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

Yes *(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

No *(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): No

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public interface UserService extends IService<User> {
 
   void fillInTeam(User user);
 
-  List<User> findByAppOwner(Long teamId);
+  List<User> listByAppOwner(Long teamId);
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/UserService.java]: how about `listByTeamId`

File: streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/AccessTokenServiceTest.java [hunk 2]
```diff
@@ -55,18 +55,18 @@ void testCrudToken() throws Exception {
     String username = JWTUtil.getUserName(jwtToken.getToken());
     Assertions.assertNotNull(username);
     Assertions.assertEquals("admin", username);
-    User user = userService.findByName(username);
+    User user = userService.getByName(username);
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/AccessTokenServiceTest.java]: how about `getByUsername` or `getByUserName` ? (case of cross-services)

**Review by RocMarshal — COMMENTED** [review]
Thanks for the improvement.
I left a few of comments, PTAL in your free time.

**Comment by VampireAchao** [comment]
Thank you for your valuable feedback and suggestions. I have addressed your comments and made the necessary changes

**Review by RocMarshal — APPROVED** [review]
Thanks for the contribution.
LGTM +1

**Pull request merged**